lib/sprig/helpers.rb

```diff
@@ -5,9 +5,12 @@ def seed_directory
     end
 
     def sprig(directive_definitions)
-      hopper = []
-      DirectiveList.new(directive_definitions).add_seeds_to_hopper(hopper)
-      Planter.new(hopper).sprig
+      plant_records(directive_definitions)
+    end
+
+    def sprig_shared(directive_definitions)
+      Sprig.configuration.seeds_directory = 'shared'
```

Comment:
This feels like a hack - I generally think of `Sprig.configuration` options as external configuration.

In this instance it looks it is being used as global state/variables.  Fair?